### ISSUE TITLE
build error on windows with custom output in generator client

### ISSUE THREAD

**Issue by vatoer** [prisma/prisma#23567]
### Bug description

build error when using custom output in generator client

```sh
Creating an optimized production build ...
glob error [Error: EPERM: operation not permitted, scandir 'C:\Users\username\Application Data'] {
  errno: -4048,
  code: 'EPERM',
  syscall: 'scandir',
  path: 'C:\\Users\\username\\Application Data'
}
Failed to compile.

Error: EPERM: operation not permitted, scandir 'C:\Users\username\Application Data'
```

build success in linux
build success if not using custom output

### How to reproduce

1. create project
2. init prisma
3. change `schema.prisma`

`src\prisma\db-1\schema.prisma`

```ts
generator client {
  provider = "prisma-client-js"
  output   = "./generated/client"
}
```

`src\prisma\db-2\schema.prisma`

```ts
generator client {
  provider = "prisma-client-js"
  output   = "./generated/client"
}
```

5. run build
```sh
pnpm build
```

### Expected behavior

build success

### Prisma information

```prisma
generator client {
  provider = "prisma-client-js"
  output   = "./generated/client"
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}
```

### Environment & setup

- OS: Windows 11
- Database: PostgreSQL
- Node.js version: v20.11.1

### Prisma Version

```
prisma                  : 5.11.0
@prisma/client          : 5.11.0
Computed binaryTarget   : windows
Operating System        : win32
Architecture            : x64
Node.js                 : v20.11.1
```

**Comment by SevInf** [comment]
Hi @vatoer 
Output you post does not look like `prisma` output. Neither to my knoweldge we try to `glob` anything during `prisma generate`. What does `pnpm build` do exactly? 
Could you provider a repo with a reporduction?

**Comment by jkomyno** [comment]
Hi @vatoer, can you please provide a reproduction [as asked above](https://github.com/prisma/prisma/issues/23567#issuecomment-2011679540)? Without that, I'm afraid we won't be able to offer much help in fixing this issue. Thanks!

**Comment by vatoer** [comment]
hello @jkomyno here is reproduction code

[poc-prisma-custom-output](https://github.com/ngcpid/poc-prisma-custom-output)

**Comment by longlongago2** [comment]
> fixed after read [#2443](https://github.com/prisma/prisma/issues/2443)
> 
> in nutshell I need to generate my output to node_modules path,简而言之，我需要将我的输出生成到 node_modules 路径下
> 
> so in my schema `src\prisma\db-1\schema.prisma`所以在我 schema `src\prisma\db-1\schema.prisma`
> 
> generator client {
>   provider = "prisma-client-js"
>   -output   = "./generated/client"
>   +output   = "../../../node_modules/@prisma-db1/client"  // modify to your node_modules path
> }
> so in my 'lib/db1.ts'所以在我 'lib/db1.ts'
> 
> import { PrismaClient } from "@prisma-db1/client";
> 
> declare global {
>   var prisma: PrismaClient | undefined;
> }
> 
> export const db1= global.prisma || new PrismaClient();
> 
> if (process.env.NODE_ENV !== "production") global.prisma = db1;

It worked for me
